### DISCUSSION TITLE
Is there a way to use mapboxgl api

### DISCUSSION THREAD

**Discussion by old-syniex** [uber/react-map-gl#1682] [Q&A] [answered]
I would like to add ScenegraphLayer from deck.gl using map.addLayer.

Is there a way to do so?

Thanks in advance

**Comment by Pessimistress** [comment, marked as answer]
https://deck.gl/docs/api-reference/mapbox/overview#using-with-react

**Reply by old-syniex** [reply]
Is this still valid with latest version? 
And is there a way to use both deckgl pickable and interactive layers on react-map-gl?

What i need is to add 3d models who is interactive over the already interactive react map
